### DISCUSSION TITLE
why it is so hard to bundle files using browserify? where are my bundled files?

### DISCUSSION THREAD

**Discussion by naveennamani** [vercel/vercel#5050] [Help] [answered]
I have a very very basic application with a single script and a single require in the index.js.
I'm using browserify to generate a bundle.js which will be used in the frontend.
I have the following files.
index.js
package.json
```json
{
..
"scripts":{
 "start":"node server.js",
 "vercel-build":"browserify index.js -o public/build.js"
 }
...
}
```
server.js - contains 5 lines of express code to serve files statically from public folder
vercel.json
```json
{
  "name": "simple-demo",
  "version": 2,
  "builds": [
    {
      "src": "public/*",
      "use": "@vercel/static"
    },
    {
      "src": "*.js",
      "use": "@vercel/node"
    }
  ],
  "routes": [
    {
      "src": "/",
      "dest": "public/index.html"
    }
  ]
}
```

I can see from log that yarn is running browserify script but no public/build.js file in generated files (source - output).
I don't know if it is generating public/build.js or not.
Or if it is ignored by the vercel?
Or am I stupid?

I've spend almost 4 hours with this simple app.
And I'm done with it for today,
Not gonna come again here.

**Comment by paulogdm** [comment, marked as answer]
Hello,

You don't need an express server to serve your static files. Also, there is no need to use a "vercel.json" for that as well. Check https://vercel.com/blog/advanced-project-settings and set the "output folder" as `public/`.

If, however, you do want a Serverless Function, then you can move "index.js" to `api/` and use [rewrites](https://vercel.com/docs/configuration#project/rewrites) for routing.

If you can share a minimal example, we will be able to guide you better.
